plugin/pulpcore/plugin/stages/declarative_version.py

```diff
@@ -12,7 +11,7 @@
 
 class DeclarativeVersion:
 
-    def __init__(self, first_stage, repository, sync_mode='mirror'):
+    def __init__(self, first_stage, repository, sync_mode):
```

Comment:
Do we want plugin writers to not have a default value for `sync_mode`?

Also if sync_mode isn't getting a default there are various docstrings that need updating.

Reply:
I deleted it because it seemed to me that the default value is assigned twice. Once here and once in the file plugin. But I didn't realize that there could be plugins that might not do that. So I made the changes back. Thanks.